Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1250,42 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      *autocommit* has three allowed values:
+
+      * ``False``: select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`~Connection.commit` and :meth:`~Connection.rollback` to
```

Comment:
```suggestion
        Use :meth:`commit` and :meth:`rollback` to
```

You're already inside `Connection`, so no need to include the class and then elide it.